Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Description: Update criterion to use 0.5.1
Forwarded: not-needed

I've raised this upstream a couple times, but upstream appears to
think that criterion 0.5.1 will bump their MSRV, and they are trying
to keep a lower MSRV.

The package builds and tests fine with criterion 0.5.1.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -72,7 +72,7 @@
 default-features = false
 
 [dev-dependencies.criterion]
-version = "0.4.0"
+version = "0.5.1"
 
 [dev-dependencies.flate2]
 version = "1.0.27"
